MAUI/DataGrid/Grouping.md

```diff
@@ -0,0 +1,572 @@
+            xmlns:syncfusion="clr-namespace:Syncfusion.Maui.DataGrid;assembly=Syncfusion.Maui.DataGrid"
+            xmlns:local="clr-namespace:GroupingUI"
+            x:Class="GroupingUI.MainPage">            
+
+    <ContentPage.BindingContext>
+        <local:ViewModel x:Name="viewModel"/>
+    </ContentPage.BindingContext>
+    
+    <StackLayout VerticalOptions="FillAndExpand">
+    
+        <Button Text="Remove Grouping"
+             TextColor="Black"
+             BackgroundColor="White"
+             Clicked="ClearGroupingButton_Click"/>
+
+        <sfgrid:SfDataGrid x:Name="dataGrid"
+                           ItemsSource="{Binding OrdersInfo}"
+                           >
+            <syncfusion:SfDataGrid.GroupColumnDescriptions>
+                <syncfusion:GroupColumnDescription ColumnName="OrderID" />
+            </syncfusion:SfDataGrid.GroupColumnDescriptions>
+            <syncfusion:SfDataGrid.GroupColumnDescriptions>
+                <syncfusion:GroupColumnDescription ColumnName="ShipCity" />
+            </syncfusion:SfDataGrid.GroupColumnDescriptions>           
+        </sfgrid:SfDataGrid>
+    
+    </StackLayout>
+
+</ContentPage>
+{% endhighlight%} 
+{% highlight c# %}
+public partial class MainPage : ContentPage
+{
+    StackLayout stackLayout;
+    SfDataGrid dataGrid;
+    ViewModel viewModel;
+    Button clearGroupingButton;
+    public MainPage()
+    {
+        InitializeComponent();
+        viewModel = new ViewModel();
+        dataGrid = new SfDataGrid();
+        clearGroupingButton = new Button();
+        stackLayout = new StackLayout();
+        dataGrid.ItemsSource = viewModel.OrdersInfo;
+        dataGrid.GroupingMode = GroupingMode.Multiple;
+        dataGrid.GroupColumnDescriptions.Add(new GroupColumnDescription()
+        {
+            ColumnName = "OrderID",
+        });
+        dataGrid.GroupColumnDescriptions.Add(new GroupColumnDescription()
+        {
+            ColumnName = "ShipCity",
+        });
+        clearGroupingButton.Text = "Remove Grouping";
+        clearGroupingButton.TextColor = Colors.Black;
+        clearGroupingButton.BackgroundColor = Colors.White;
+        clearGroupingButton.Clicked += ClearGroupingButton_Click;
+        stackLayout.Children.Add(clearGroupingButton);
+        stackLayout.Children.Add(dataGrid);
+        this.Content = stackLayout;       
+    }
+}
+{% endhighlight%} 
+{% endtabs %}
+
+{% highlight c# %}
+private void ClearGroupingButton_Click(object sender, System.EventArgs e)
+{
+    //Clearing the Group
+    //dataGrid.GroupColumnDescriptions.Clear();
+
+    //Removing the Group based on group item
+    var groupColumn = dataGrid.GroupColumnDescriptions[1];
+    dataGrid.GroupColumnDescriptions.Remove(groupColumn);
+        
+    //Removing the Group based on group index
+    //dataGrid.GroupColumnDescriptions.RemoveAt(0);
+}
+{% endhighlight%} 
+
+Run the application to render the following output: 
```

Comment:
Images are not required for this topic. Users can easily understand the concept.

Reply:
Had removed the Images.